src/Languages/简体中文 (Simplified Chinese).json

```diff
@@ -12,55 +12,54 @@
   "portal2_config_button": "Portal 2 配置",
   "dev_settings_button": "开发者设置",
   "dev_mode_enabled": "开发者模式已启用！",
-  "back_button": "回到。",
+  "back_button": "返回",
   "mount_button": "安装",
-  "mounted": "安装完成！",
+  "mounted": "已安装！",
   "mount_complete": "安装完成！",
-  "mounting_mod": "安装模块！",
+  "mounting_mod": "正在安装！",
   "unmount_button": "卸载",
-  "unmounted_toast": "未装载!",
-  "args-error": "启动参数解析失败！\n仅使用默认启动参数！\n请查看日志获取更多信息！",
```

Comment:
It seems you accidentally removed this translation

Reply:
The "args-error" one to be more specific

Reply:
I didn't see 'args-error' in english.json, so I removed it.

Reply:
Its right here https://github.com/Portal-2-Multiplayer-Mod/Portal-2-Multiplayer-Mod/blob/693c664954536ec9e5f29c886fa0395aaa75ecc9/src/Languages/English.json#L22

Reply:
Oh, my fault. I used the English.json from the main branch to compare. It's fixed now.